static/app/components/events/featureFlags/featureFlagOnboardingSidebar.tsx

```diff
@@ -146,6 +150,10 @@ function OnboardingContent({
   hasDocs: boolean;
 }) {
   const organization = useOrganization();
+  const ORIGINAL_HASH = useMemo(() => {
+    return window.location.hash;
+  }, []);
```

Comment:
Why does this need a `useMemo`?

Reply:
Can we leave a comment? that's weird it changes to unset -- are we changing the hash in this component?

Reply:
we aren't changing it at all ... i couldn't figure out why it was happening😕